src/shapes.rs

```diff
@@ -86,98 +107,23 @@ pub fn draw_rectangle_lines(x: f32, y: f32, w: f32, h: f32, thickness: f32, colo
     draw_rectangle(x, y + t, t, h - t, color);
 }
 
-pub fn draw_hexagon(
```

Comment:
I would like to keep `draw_hexagon` as a special case of `draw_poly`. For hex-based games :) 
And maybe later `draw_hexagon` can be extended with some additional hexagon-specific params